.bazelrc

```diff
@@ -0,0 +1,18 @@
+# Enable Bzlmod for every Bazel command
+common --enable_bzlmod
+
+common --remote_cache=grpcs://remote.buildbuddy.io
```

Comment:
Do you mind creating us a buildbuddy account? They provide their services for open source projects

Reply:
Already created, we can setup the stuff in a different PR.

Reply:
> Already created, we can setup the stuff in a different PR.

Perfect!